pdfarranger/pdfarranger.py

```diff
@@ -429,7 +429,7 @@ def do_activate(self):
         self.__create_actions()
 
     @staticmethod
-    def set_cellrenderer_data(column, cell, model, it, data=None):
+    def set_cellrenderer_data(_column, cell, model, it, _data=None):
```

Comment:
We need those =None for functions which get called not only from Gtk code but also from our own code.